kythe/go/util/vnameutil/BUILD

```diff
@@ -1,3 +1,4 @@
+# gazelle:go_test file
```

Comment:
What are these directives?

Reply:
These are the directives added by the Gazelle patch to support per-file test generation, rather than per-package.